src/main/controllers/helpers/CopyToOtherPartyHelper.ts

```diff
@@ -4,18 +4,19 @@ import { PageUrls, Rule92Types } from '../../definitions/constants';
 import { AnyRecord } from '../../definitions/util-types';
 
 export const getCaptionTextForCopyToOtherParty = (req: AppRequest, translations: AnyRecord): string => {
+  let captionText = '';
   const contactType = req.session.contactType;
   if (contactType === Rule92Types.CONTACT) {
     const captionSubject = req.session.userCase.contactApplicationType;
-    return translations.sections[captionSubject]?.caption;
+    captionText = translations.sections[captionSubject]?.caption;
   }
   if (contactType === Rule92Types.RESPOND) {
-    return translations.respondToApplication;
+    captionText = translations.respondToApplication;
```

Comment:
Seemed better before no? Now you're evaluating each if condition every time. Unless that was intended? I would revert or replace with else ifs

Reply:
I updated this as referring to the logic in CopyToOtherPartyController:
https://github.com/hmcts/et-sya-frontend/blob/master/src/main/controllers/CopyToOtherPartyController.ts#L95
Happy to use "return" if it is safe to do so